src/tests/class_tests/openms/source/AAIndex_test.cpp

```diff
@@ -1,306 +1,413 @@
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('I'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('L'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('K'),1.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('M'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('F'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('P'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('S'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('T'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('W'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('Y'),0.)
-  TEST_REAL_SIMILAR(AAIndex::getFAUJ880111('V'),0.)
-END_SECTION
-
-START_SECTION(static double getFINA770101(char aa))
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('A'),1.08)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('R'),1.05)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('N'),0.85)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('D'),0.85)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('C'),0.95)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('Q'),0.95)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('E'),1.15)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('G'),0.55)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('H'),1.00)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('I'),1.05)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('L'),1.25)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('K'),1.15)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('M'),1.15)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('F'),1.10)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('P'),0.71)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('S'),0.75)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('T'),0.75)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('W'),1.10)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('Y'),1.10)
-  TEST_REAL_SIMILAR(AAIndex::getFINA770101('V'),0.95)
-END_SECTION
-
-START_SECTION(static double getARGP820102(char aa))
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('A'),1.18)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('R'),0.20)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('N'),0.23)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('D'),0.05)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('C'),1.89)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('Q'),0.72)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('E'),0.11)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('G'),0.49)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('H'),0.31)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('I'),1.45)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('L'),3.23)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('K'),0.06)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('M'),2.67)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('F'),1.96)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('P'),0.76)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('S'),0.97)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('T'),0.84)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('W'),0.77)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('Y'),0.39)
-  TEST_REAL_SIMILAR(AAIndex::getARGP820102('V'),1.08)
-END_SECTION
-
-
-END_TEST
+class inner_Node
```

Comment:
should be `InnerNode`
```suggestion
class InnerNode
```